Source/ASConfigurationInternal.mm

```diff
@@ -103,12 +99,12 @@ + (void)test_resetWithConfiguration:(ASConfiguration *)configuration
 
 @end
 
-BOOL ASActivateExperimentalFeature(ASExperimentalFeatures feature)
+BOOL _ASActivateExperimentalFeature(ASExperimentalFeatures feature)
 {
-  return [ASGetSharedConfigMgr() activateExperimentalFeature:feature];
+  return [ASConfigurationManagerGet() activateExperimentalFeature:feature];
```

Comment:
Just a nit: Also we may should rename `activateExperimentalFeature:` to `activateExperimentalFeatures:`.